requirements_dev.txt

```diff
@@ -1,8 +1,9 @@
 aiohttp~=3.9
 mypy>=1.8.0
 pre-commit>=3.6.1
+pytest-asyncio==0.23.5
```

Comment:
are those explicit or sub-dependencies from pytest?

Reply:
explicit, required to run async funtions in pytest